src/System.Runtime.Serialization.Json/tests/ReflectionOnly/Resources/System.Runtime.Serialization.Json.ReflectionOnly.Tests.rd.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
+  <Library Name="*System.Private.DataContractSerialization*">
+    <Assembly Name="System.Private.DataContractSerialization" Dynamic="Required All">
```

Comment:
@shmao , I'd be worried that if these tests need this that we're hiding a product problem.

@tijoytom , can you please elaborate on what made you write this? Is the library reflecting on itself or are the tests doing their own reflection that requires this?